_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,264 @@
+
+```html
+<table>
+  <thead>	
+    <tr>
+      <td role='columnheader' id="header1">Projects</td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td headers="header1">15%</td>
+    </tr>
+  </tbody>
+</table>
+```
+
+#### Passed Example 4
+
+A `table` with multiple `columnheader(s)`, where the `headers` attribute on the cell lists the `id(s)` of `th` within the same `table`.
+
+```html
+<table>
+  <tr>
+    <th colspan="2" id="header1">Projects</th>
+    <th colspan="2" id="header2">Exams</th>
+  </tr>
+  <tr>
+    <th id="e1" headers="header1">1</th>
+    <th id="e2" headers="header1">2</th>
+    <th id="p1" headers="header2">1</th>
+    <th id="p2" headers="header2">2</th>
+  </tr>
+  <tr>
+    <td colspan="2" headers="header1 e1 e2">15%</td>
+    <td headers="header2 p1">15%</td>
+    <td headers="header2 p2">45%</td>
+  </tr>
+</table>
+```
+
+### Failed
+
+#### Failed Example 1
+
+Cells with `headers` attribute, refers to non-existing `th` within the same `table`.
+
+```html
+<table>
+  <tr>
+    <th id="header1">Projects</th>
+    <th id="header2">Exams</th>
+  </tr>
+  <tr>
+    <td headers="header3">15%</td>
+    <td headers="header3">15%</td>
+  </tr>
+</table>
+```
+
+#### Failed Example 2
+
+One of the cells with `headers` attribute, refers to `th` that does not exist within the same `table`.
+
+```html
+<table>
+  <tr>
+    <th colspan="2" id="header1">Projects</th>
+  </tr>
+  <tr>
+    <th id="e1" headers="header1">1</th>
+    <th id="e2" headers="header1">2</th>
+  </tr>
+  <tr>
+    <td colspan="2" headers="header1 e1 NOT-EXIST">15%</td>
+  </tr>
+</table>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+There's no usage of the `headers` attribute.
+
+```html
+<table>
+  <tr>
+    <th scope="col">Projects</th>
+    <th scope="col">Exams</th>
+  </tr>
+  <tr>
+    <td>15%</td>
+    <td>45%</td>
+  </tr>
+</table>
+```
+
+#### Inapplicable Example 2
+
+A table used for presentation only, that has a `role="presentation"`. 
+
+```html
+<table role="presentation">
+  <tr>
+    <th colspan="2">My document title</th>
+  </tr>
+  <tr>
+    <td><p>The excerpt</p></td>
+    <td><p>The content</p></td>
+  </tr>
+</table>
+```
+
+#### Inapplicable Example 3
+
+The rule applies only to `table` element.
+
+```html
+<div role="table">
+  <div role="row">
+    <div class="columnheader" id="header1">Projects</div>
+    <div class="columnheader" id="header2">Exams</div>
+  </div>
+  <div role="row">
+    <div role="cell" headers="header2">15%</div>
+    <div role="cell" headers="header1">15%</div>
+  </div>
+</div>
+```
+
+#### Inapplicable Example 4
+
+No `headers` attribute is defined for table `cell`.
+
+```html
+<table>
+  <tr>
+    <th id="header1">Projects</th>
+    <th id="header2">Exams</th>
+  </tr>
+  <tr>
+    <td colspan="2">15%</td>
+  </tr>
+</table>
+```
+
+#### Inapplicable Example 5
+
+A table that is not visible. 
```

Comment:
There is nothing in the applicability about visibility. Not included in the accessibility is covered in the next test case, so you don't need to duplicate.